dev/com.ibm.ws.jaxrs.2.x.monitor/src/com/ibm/ws/jaxrs/monitor/JaxRsMonitorProviderRegister.java

```diff
@@ -0,0 +1,77 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.jaxrs.monitor;
+
+import java.util.List;
+import java.util.Set;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.ConfigurationPolicy;
+
+import com.ibm.ws.container.service.app.deploy.ApplicationInfo;
+import com.ibm.ws.container.service.state.ApplicationStateListener;
+import com.ibm.ws.jaxrs.monitor.JaxRsMonitorFilter.RestfulMetricInfo;
+import com.ibm.ws.jaxrs20.providers.api.JaxRsProviderRegister;
+
+@Component(immediate = true, property = { "service.vendor=IBM" }, configurationPolicy = ConfigurationPolicy.IGNORE, service = {JaxRsProviderRegister.class, ApplicationStateListener.class})
+public class JaxRsMonitorProviderRegister implements JaxRsProviderRegister, ApplicationStateListener {
+
+	private JaxRsMonitorFilter monitorFilter = new JaxRsMonitorFilter();
+	
+    @Override
+    public void installProvider(boolean clientSide, List<Object> providers, Set<String> features) {
+        
+    	// Register the metrics monitor filter class if we are not on the client and if using the monitor
+    	// feature with or without the mpMetrics feature.
+        if (!clientSide) {
+            if (features.contains("monitor-1.0")) {
```

Comment:
We can delete this check now.  For beta, this feature should always pull in monitor-1.0.  For GA, it will be an auto-feature that is only enabled if monitor-1.0 is enabled, so we shouldn't waste time checking for it.  If this method is called on the server (not client), then we should add the monitorFilter.

Reply:
Done.